changelogs/unreleased/gh-7086-box_issue_promote_demote_asserts.md

```diff
@@ -0,0 +1,5 @@
+## bugfix/synchro
+
+* Fixed asserts in debug build/undefined behaviour in release when simulteneous
+  elections started/another instance promoted while instance was acquiring
+  or releasing limbo (gh-7086).
```

Comment:
We try to call limbo the "synchro queue" when speaking to the public.

Reply:
Fixed